packages/twenty-server/src/modules/workflow/common/query-hooks/workflow-version/utils/assert-workflow-version-trigger-is-defined.util.ts

```diff
@@ -0,0 +1,22 @@
+import { WorkflowVersionWorkspaceEntity } from 'src/modules/workflow/common/standard-objects/workflow-version.workspace-entity';
+import { WorkflowTrigger } from 'src/modules/workflow/common/types/workflow-trigger.type';
+import {
+  WorkflowTriggerException,
+  WorkflowTriggerExceptionCode,
+} from 'src/modules/workflow/workflow-trigger/workflow-trigger.exception';
+
+export function assertWorkflowVersionTriggerIsDefined(
```

Comment:
we suffix our utils that throw with orThrow

Reply:
See https://github.com/twentyhq/twenty/pull/6926#discussion_r1754662374